packages/orbit-components/src/Itinerary/ItineraryPart/components/ItineraryPartStatus/index.js

```diff
@@ -0,0 +1,104 @@
+// @flow
```

Comment:
Since all components in `*/ItineraryPart/components/*` are public API, I would suggest to have it as in folder structure like:`Itinerary/ItineraryPart/ItineraryPartStatus`. So it's aligned with the rest of our code base and API.

Components in `*/components` folder are being used only as internal one – for instance check `Card` component.